index.html

```diff
@@ -242,40 +191,41 @@ <h2>Photo Gallery</h2>
     <section class="survey">
         <h2>Survey</h2>
         <form method="POST" action="https://formspree.io/xvogdwnk">
-            <label>What is your name?<br>
-                <input name="name_field" type="text" required>
-            </label>
-            <label>What is your email?<br>
-                <input name="email_field" type="email" required>
-            </label>
```

Comment:
You were supposed to use the `for` attribute for all labels except the radio buttons.  You did this backwards